problemsets-mat223/modules/module8-exercises.tex

```diff
@@ -59,6 +104,16 @@
 				basis $\mathcal N$ for $\R^{3}$ such that
 				$[\vec b]_{\mathcal N}=\mat{0\\1\\2}$.
 		\end{enumerate}
+		\begin{solution}
+			\begin{enumerate}
+				\item Let $\mathcal{M} = \{ \vec{u}, \vec{v}\}$. Then $[\vec{a}]_{\mathcal{M}} = \mat{1\\0} \Rightarrow \vec{a} = 1 \vec{u} + 0 
```

Comment:
Use $\Set{..}$ (fix this throughout). And, no `\Rightarrow`. Use the word "implies" instead.

Reply:
What do I replace when I use $\Set{..}$ ?

Reply:
` \{ \vec{u}, \vec{v}\}` -> ` \Set{ \vec{u}, \vec{v}}`, for example. The set braces should never be done manually. They should always be done with the `\Set{}` command